legal-api/src/legal_api/resources/v2/internal_services.py

```diff
@@ -36,6 +36,15 @@ def get_future_effective_filing_ids():
     return jsonify(filing_ids), HTTPStatus.OK
 
 
+@bp.route('/expired_restoration', methods=['GET'])
```

Comment:
@AimeeGao use this endpoint to get the expired restoration business

Reply:
Many thanks, Vysakh (^-^)